tests.cpp

```diff
@@ -243,23 +226,6 @@ TEST_CASE_METHOD(CoherentUnitTestFixture, "test_xor")
     REQUIRE_THAT(qftReg, HasProbability(0xd3e));
 }
 
-TEST_CASE_METHOD(CoherentUnitTestFixture, "test_m_reg")
```

Comment:
Did this test have any redeeming value?  Doesn't look like it, reading it now...

Reply:
Maybe not. I think it was at least redundant with another test.